dev-libs/spdlog/files/spdlog-unbundle-fmt.patch

```diff
@@ -0,0 +1,90 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
To be honest, I don't really like the idea of having to carry around such a big patch forever. Would be nice to integrate it upstream, e.g. as CMake option.